packages/client/src/pages/home/map.tsx

```diff
@@ -84,9 +85,12 @@ const Map = () => {
     lat: location.lat,
     lng: location.lon,
   });
+
   const [name, setName] = React.useState(location.name);
 
   if (initialLocation === undefined) {
+    // TODO: this block is causing getPostsByFilter to be called twice when the page is refreshed
```

Comment:
Should we just keep lat/lon in the store rather than both?

Reply:
I wanted to but I had an issue with needing the map to update the post-list when I tried keeping it in the store :monkey_face: